cpp/include/bindings/bindings_renderer.hpp

```diff
@@ -122,16 +122,16 @@ extern "C"
 
 namespace rive
 {
-	RenderPaint* makeRenderPaint()
-	{
-		assert(g_JNIRenderer != nullptr);
-		return g_JNIRenderer->makeRenderPaint();
-	}
-
-	RenderPath* makeRenderPath()
-	{
-		assert(g_JNIRenderer != nullptr);
-		return g_JNIRenderer->makeRenderPath();
-	}
+	// RenderPaint* makeRenderPaint()
+	// {
+	// 	assert(g_JNIRenderer != nullptr);
+	// 	return g_JNIRenderer->makeRenderPaint();
+	// }
+
+	// RenderPath* makeRenderPath()
+	// {
+	// 	assert(g_JNIRenderer != nullptr);
+	// 	return g_JNIRenderer->makeRenderPath();
+	// }
```

Comment:
Oh! This is all gone now? How does this work when loading files then?

Reply:
Oh! So that's taking over everything, sweet! We can probably remove all of this altogether. Also the g_JNIRenderer?